### PR TITLE
Updated metric_input_measure to reflect correctly on what it can be

### PR DESCRIPTION
**Pull request by WilliamDee**
## Context
In dbt and in SL, the definition of the metric_input_measure can be either a string or an object. There is some logic to convert the string to an object if that is provided. However, in the schema it shows that it can only be an object. This is causing warnings to show up on the IDE even though it is parseable.

image

### CODE REVIEW

**Review by dave-connors-3 — APPROVED** [review]
if you wouldn't mind updating the tests in the `tests/` directory, that would be swell!

**Comment by WilliamDee** [comment]
> if you wouldn't mind updating the tests in the `tests/latest` directory, that would be swell! effectively all you need to do is add valid yml related to your change to relevant file in `tests/latest/valid` and CI should ensure it's valid!

@dave-connors-3 Done! Seems like pushing that commit dismissed your approval. Mind taking a look again? Also, I know this is reflected on the IDE, after I merge this, what's the process/timeline for this being reflected in prod IDE? Thank you 🙏

**Review by dave-connors-3 — APPROVED** [review]

**Pull request merged**
